Source/Config/Mixins/Templates/RadioButtonGroup.lua

```diff
@@ -0,0 +1,62 @@
+AuctionatorConfigRadioButtonGroupMixin = {}
+
+function AuctionatorConfigRadioButtonGroupMixin:OnLoad()
+  if self.groupHeadingText ~= nil then
+    self.GroupHeading.subHeadingText = self.groupHeadingText
+  end
+
+  self.radioButtons = {}
+  self:SetupRadioButtons()
+  self.radioButtonGroupOnChangeEvent = function() end
+end
+
+function AuctionatorConfigRadioButtonGroupMixin:SetupRadioButtons()
+  local children = { self:GetChildren() }
+  local size = 0
+
+  for _, child in ipairs(children) do
+    if child.isAuctionatorRadio then
+      table.insert(self.radioButtons, child)
+
+      child:SetPoint("TOPLEFT", 0, size * -1)
+      child.RadioButton.Label:SetPoint("TOPLEFT", 20, -2)
+
+      child.onSelectedCallback = function()
+        self:RadioSelected(child)
+      end
+    end
+
+    size = size + (child:GetHeight() or 20)
+  end
+
+  -- 8 is for bottom padding
+  self:SetHeight(size + 8)
```

Comment:
You're not wrong, but its a constraint of the XML/LUA hybrid, unfortunately. For most components, the anchoring is done directly in the XML, and the "20" is used to create our left padding. We *could* take it out, but we would not be able to namespace it in order to include it here; it would have to be a global constant. Since the radio buttons must be anchored dynamically in this component, I need the 20 here to make it consistent with the 20s in the XML. Seems overkill to constantize that (and the 8 referenced in this comment) for a single use. 

I cry "magic number" when a number appears multiple times in the codebase, and any transpositional or other error with the number will affect the logic. Pixel pushing, especially when the number is only used once, doesn't normally catch my attention.

That's my thinking, happy to constantize if you feel like it would still be of benefit!